$MirOS: ports/www/lynx/patches/patch-configure_in,v 1.1 2008/12/17 00:24:41 tg Exp $

	Fix -Werror builds

--- configure.in.orig	Sun Jan 18 19:41:33 2009
+++ configure.in	Sun Jan 18 19:50:15 2009
@@ -189,7 +189,7 @@ AC_MSG_RESULT($with_warnings)
 if test "$with_warnings" = "yes"
 then
 	CF_GCC_ATTRIBUTES
-	CF_GCC_WARNINGS
+	CF_GCC_WARNINGS(Wno-cast-qual Wno-cast-align)
 fi
 fi
 
